lib/shelley/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -172,25 +174,40 @@ instance TxWitnessTagFor IcarusKey where
 instance TxWitnessTagFor ByronKey where
     txWitnessTagFor = TxWitnessByronUTxO Byron
 
+
+-- | Returns a tuple of unsigned transactions and withdrawals.
+mkTxUnsigned
+    :: Cardano.NetworkId
+    -> [Cardano.Certificate]
+    -> Maybe Cardano.TxMetadata
+    -> SlotNo
+    -- ^ Time to Live
+    -> XPrv
+    -- ^ Reward account
+    -> CoinSelection
+    -> (Cardano.TxBody Cardano.Shelley, [(Cardano.StakeAddress, Cardano.Lovelace)])
+mkTxUnsigned networkId certs md timeToLive rewardAcnt cs =
+    let wdrls = mkWithdrawals
+            networkId
+            (toChimericAccountRaw . toXPub $ rewardAcnt)
+            (withdrawal cs)
+        unsigned = mkUnsignedTx timeToLive cs md wdrls certs
+    in (unsigned, wdrls)
```

Comment:
What does this function buys us exactly :thinking: ?

Reply:
It's quite neat :smile: 

I believe the idea was originally to re-use it somewhere, but we don't...